.circleci/config.yml

```diff
@@ -294,6 +295,28 @@ workflows:
                 - beta
                 - alpha
 
+  publish_to_docs_site:
+    jobs:
+      - publish-docs/publish_docs:
```

Comment:
There is also a build_docs workflow on the documentation site, is that needed here as well?

Reply:
That might be a question for @trevor-coleman. I'm unsure if this CI script is set up correctly. This branch was made a few months ago and things might have changed with the CI setup.

Reply:
build-docs is "only build" -- good for automated testing so PRs don't get merged with broken links etc.
publish-docs is "build and publish" -- so it builds it, and pushes it to `ir-docs`